internal/cli/atlas/deployments/search/indexes/create_test.go

```diff
@@ -126,6 +127,7 @@ func TestCreate_RunLocal(t *testing.T) {
 		},
 		Name:           opts.Name,
 		SearchAnalyzer: &opts.SearchAnalyzer,
+		Type:           pointer.Get(search.DefaultType),
```

Comment:
no need for the helper
```suggestion
		Type:           &search.DefaultType,
```

Reply:
because DefaultType is a const and not a var, &search.DefaultType will not:
`invalid operation: cannot take address of search.DefaultType (untyped string constant "search")`
I would prefer not making this a `var`, wdyt?

Reply:
I have fear for for the pointer helper leaking across the codebase when it's mostly intended for tests, for const you can usually do `temp := search.DefaultType; &temp` but I can understand is overkill so no changes needed